opm/autodiff/StandardWellsDense_impl.hpp

```diff
@@ -249,13 +252,51 @@ namespace Opm {
     }
 
 
+    template<typename FluidSystem, typename BlackoilIndices, typename ElementContext, typename MaterialLaw>
+    template <typename Simulator>
+    void
+    StandardWellsDense<FluidSystem, BlackoilIndices, ElementContext, MaterialLaw>::
+    getMobility(const Simulator& ebosSimulator, const int perf, const int cell_idx, std::vector<EvalWell>& mob) const
+    {
+
+        const int np = wells().number_of_phases;
+        assert (mob.size() == np);
+        const auto& intQuants = *(ebosSimulator.model().cachedIntensiveQuantities(cell_idx, /*timeIdx=*/0));
```

Comment:
I'm sure you have tested this and that it works correctly, but we are trying to remove explicit access to the cache. I am not going to require that now, but would appreciate if @andlaus could take a brief look at the cache usage here.